misk-grpc-reflect/build.gradle.kts

```diff
@@ -32,7 +32,7 @@ dependencies {
   implementation(libs.kotlinLogging)
   implementation(libs.kotlinReflect)
   implementation(libs.okio)
-  implementation(libs.wireGrpcClient)
+//  implementation(libs.wireGrpcClient)
```

Comment:
Should we just remove these?

Reply:
yes for sure. I should have done that before converting from draft to ready for review.

Is there any documentation on how to update the ABI baselines?

Reply:
I think you can run `gradle apiDump` and commit the changed `.api` files

Reply:
The contributing guide should have more info here: https://github.com/cashapp/misk/blob/master/CONTRIBUTING.md#breaking-changes